src/app/components/alert/alert.component.scss

```diff
@@ -16,8 +16,17 @@
  *   If not, see https://www.gnu.org/licenses/lgpl-3.0.txt
  *
  */
+@import "variables";
+
 ::ng-deep .alert {
-    margin: 0 0 10px 0;
-    box-shadow: 0 3px 6px #0000001a;
-    pointer-events: all;
+  max-width: 600px;
+  width: 100%;
+  margin: 0 auto 10px auto;
+  box-shadow: $box-shadow;
+  pointer-events: all;
+
+  @media only screen and (max-width: 600px) {
+    max-width: 60%;
```

Comment:
Why set the max-width to 60%? 
![image](https://user-images.githubusercontent.com/55654104/114552235-a98a2e00-9c64-11eb-803c-b7dd06829fe4.png)
And adding margin would improve the experience as well in my opinion.